### PR TITLE
CLEAN(I/O) Remove StreamToPDU and posix-read

### PR DESCRIPTION
**Pull request by koolfunky**
The streamToPDU function was doing I/O but in the library it could be
better to not having I/O.
The I/O should be done by the library user.

### CODE REVIEW

**Comment by michael-zapata** [comment]
Glad to see this coming to fruitition!

File: lib/kinetic.js [hunk 1]
```diff
@@ -485,51 +484,13 @@ class PDU extends stream.Readable {
             }
         }
 
+
```
Comment (michael-zapata) [inline lib/kinetic.js]: Nit: No need for this line here.

**Review by ghost — CHANGES_REQUESTED** [review]
This is great!

But we should also edit the documentation accordingly (README.md).

**Comment by AntoninCoulibaly** [comment]
@michael-zapata ok I'm on it. it will be in a separate PR

**Review by MathieuCassagne — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by michael-zapata** [comment]
Just add an issue when you merge this one, so that we're clear the doc is outdated.

**Comment by AntoninCoulibaly** [comment]
@michael-zapata  perfect thank you thank you. I'm adding it to the project

**Pull request merged**